app/views/user_journey/upload_certificate.html.erb

```diff
@@ -11,25 +11,48 @@
       </legend>
       <div class="govuk-radios govuk-radios--conditional govuk-!-margin-bottom-5" data-module="govuk-radios">
         <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="upload-certificate-conditional" name="upload-certificate" type="radio" value="file" data-aria-controls="conditional-upload-certificate-conditional">
+          <input class="govuk-radios__input" id="upload-certificate-conditional" name="upload-certificate"
+            type="radio" value="file" data-aria-controls="conditional-upload-certificate-conditional"
+            <%=
+              if @upload.errors.key?(:certificate) && params['upload-certificate'] == 'file'
+                checked='checked'
+              end
+            %>
+          >
           <label class="govuk-label govuk-radios__label" for="upload-certificate-conditional">
             <%= t 'user_journey.certificate.upload_certificate_file_header' %>
           </label>
         </div>
         <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-upload-certificate-conditional">
           <div class="govuk-form-group">
-            <%= f.file_field :cert_file, class: "govuk-file-upload" %>
+            <% if @upload.errors.key?(:certificate) && params['upload-certificate'] == 'file' %>
```

Comment:
could we move the conditional logic the controller? too much duplication and logic within the view.. we could then do something like
view:
`<%= f.file_field :cert_file, class: "govuk-file-upload #{error_class('file')}" %>`

controller:
```
def error_class(type)
    return '' unless @upload.errors.key?(:certificate)

    if type == 'file'
       'govuk-file-upload--error'
    else
       'govuk-textarea--error'
    end
end
```